Sources/Core/ObjectiveCDictionaryExtension.swift

```diff
@@ -108,14 +104,12 @@ extension ObjCFileRenderer {
                 case .string(format: .some(.uri)):
                     return "[\(destArray) addObject:[\(processObject) absoluteString] ];"
                 case .string(format: .some(.dateTime)):
-                    return [
-                        ObjCIR.ifStmt("\(propIVarName) != nil && [NSValueTransformer allowsReverseTransformation]") {[
+                    return
+                        ObjCIR.ifElseStmt("\(propIVarName) != nil && [NSValueTransformer allowsReverseTransformation]") {[
```

Comment:
I don't think I saw this the first time, don't we have to get the actual `NSValueTransformer` subclass here?

Reply:
We should be able to get the transformer instance with `dateValueTransformerKey`, get it's class and then ask it if it `allowsReverseTransformation`